documentation/writing-policies-mutate.md

```diff
@@ -138,6 +138,82 @@ spec:
 
 Note, that if **remove** operation cannot be applied, then this **remove** operation will be skipped with no error.
 
+
+## Patch
```

Comment:
As `mutate.patch` will be deprecated and replace by `mutate.patchesJson6902`, do we need to add the doc here?

Reply:
okay. I will remove it.